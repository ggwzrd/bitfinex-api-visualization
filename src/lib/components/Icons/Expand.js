import React from "react"
import Icon from "./Icon"

const title = "Expand"
const path = (<path d="M12,8 L12,7 L17,7 L17,12 L16,12 L16,9 L13,12 L12,11 L15,8 L12,8 Z M12,16 L12,17 L7,17 L7,12 L8,12 L8,15 L11,12 L12,13 L9,16 L12,16 Z" />)

const Expand = props => (
    <Icon {...props} title={title} path={path} />
)

export default Expand
