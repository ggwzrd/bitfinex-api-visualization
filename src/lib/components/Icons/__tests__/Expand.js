import React from "react"
import createShallow from "@material-ui/core/test-utils/createShallow"
import Expand from "../Expand"

describe("<Expand />", () => {
    it("Does not explode", () => {
        const element = createShallow()(<Expand />)
        expect(element).toMatchSnapshot()
    })
})
