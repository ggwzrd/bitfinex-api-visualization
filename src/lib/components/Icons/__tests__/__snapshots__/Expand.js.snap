// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Expand /> Does not explode 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Expand />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "className": "",
      "color": "#000000",
      "height": 24,
      "path": <path
        d="M12,8 L12,7 L17,7 L17,12 L16,12 L16,9 L13,12 L12,11 L15,8 L12,8 Z M12,16 L12,17 L7,17 L7,12 L8,12 L8,15 L11,12 L12,13 L9,16 L12,16 Z"
/>,
      "title": "Expand",
      "width": 24,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "className": "",
        "color": "#000000",
        "height": 24,
        "path": <path
          d="M12,8 L12,7 L17,7 L17,12 L16,12 L16,9 L13,12 L12,11 L15,8 L12,8 Z M12,16 L12,17 L7,17 L7,12 L8,12 L8,15 L11,12 L12,13 L9,16 L12,16 Z"
/>,
        "title": "Expand",
        "width": 24,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "context": Object {},
  },
}
`;
